test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -202,15 +290,13 @@ func assertNumEntries(cacheName, pod string, numEntries int, client ispnclient.H
 		panic(fmt.Sprintf("Unexpected response code %d", rsp.StatusCode))
 	}
 
-	/*
```

Comment:
@ryanemerson do you need this commented code fragment?

Reply:
I'm not sure what you mean? In my changes I have uncommented the code.

Reply:
Oh, sorry, was wrong to see.